### PR TITLE
Updating Kompact benches to use tell_serialised() interface

### PR DESCRIPTION
**Pull request by adamhass**
Had to bump synchronoise dependencies in Actix as well. 

I've left the explicit initialise_pool() calls in the actors because I believe it's better to initialise them during set-up phase since we don't want to benchmark the buffer pool init...

### CODE REVIEW

File: runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ object Runner {
   type Stub = BenchmarkRunnerGrpc.BenchmarkRunnerStub;
 
   val WAIT: Duration = 500.milliseconds;
-  val MAX_RETRIES: Int = 30;
+  val MAX_RETRIES: Int = 130;
```
Comment (Bathtor) [inline runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala:19]: What's with that? Do you really want to wait 2min before the runner realises that some node can't be connected to?

**Comment by Bathtor** [comment]
This is still for Scala 2.12 Ammonite, right? What happened to the other changes you did for Scala 2.13?

File: runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala (hunk 1 above)
Comment (adamhass) [reply runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala:19]: 30*500ms was not enough for some benchmarks to init properly. My most recent run had a benh retry 45 times before succeeding, on my workstation, so 30 is definitely too low.

Comment (Bathtor) [reply runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala:19]: Really? Gee, that's very long.

**Comment by adamhass** [comment]
I still have changes for Scala 2.13 on a different branch but I was having too many issues I figured I'd merge the Kompact changes before I resolve all the Scala things so Harald can do his thing with Kompact.

We can put this PR on hold until I've resolved the scala stuff if you want. It's taking a while for me to figure out all the different versions and dependencies and how to keep it neat but I think I can make it work.

**Comment by Bathtor** [comment]
> I still have changes for Scala 2.13 on a different branch but I was having too many issues I figured I'd merge the Kompact changes before I resolve all the Scala things so Harald can do his thing with Kompact.
> 
> We can put this PR on hold until I've resolved the scala stuff if you want. It's taking a while for me to figure out all the different versions and dependencies and how to keep it neat but I think I can make it work.

Hmm, since it'll hold up both your theses for something fairly unrelated, let's not wait for that now. But we shouldn't forget about once you defended. It's slightly annoying having to hold back newer Ammonite versions all the time. It would be helpful if Ammonite were like the python interpreter, where you can multiple versions installed and the header of the file tells the system which one to load...

File: runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala (hunk 1 above)
Comment (haraldng) [reply runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala:19]: Same for me, sometimes mine starts to run on 120th attempt...

**Pull request merged**